### PR TITLE
chore: upgrade README

### PR DESCRIPTION
**Pull request by samlfair**
## Context

This updates the README.md, which is out of date.

## Checklist before requesting a review
- [x] I hereby declare my code ready for review.
- [ ] If it is a critical feature, I have added tests.
- [ ] The CI is successful.
- [ ] If there could backward compatibility issues, it has been discussed and planned.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,91 +16,52 @@
...
-
-
----
+Create a Next.js or Nuxt app:
+```
+npx create-nuxt-app my-app
+```
+```
+npx nuxi@latest init my-app
+```
```
Comment (lihbr) [inline README.md]: Those are only Nuxt, is it normal?

**Review by lihbr — APPROVED** [review]
LGTM, maybe a small typo thing to review?

File: README.md (hunk 1 above)
Comment (samlfair) [reply README.md]: Haha, nope. The first one should be next. I've fixed it.

**Review by hypervillain — APPROVED** [review]
👍

**Pull request merged**